javascript/ql/src/Security/CWE-843/TypeConfusionThroughParameterTampering.qhelp

```diff
@@ -35,6 +35,15 @@
 			is user-controlled.
 
 		</p>
+
+		<p>
+
+			An even safer alternative is to design the application such that sanitization isn't
+			needed at all, for instance by using prepared statements for SQL queries.
```

Comment:
```suggestion
			An even safer alternative is to design the application so that sanitization is not
			needed, for instance by using prepared statements for SQL queries.
```